src/app/Http/Controllers/PostController.php

```diff
@@ -6,25 +6,26 @@
 use Illuminate\View\View;
 use App\Services\PostService;
 use App\Http\Requests\PostRequest;
+use App\Models\Photo;
 use Illuminate\Http\RedirectResponse;
 
 class PostController extends Controller
 {
     public $postService;
-    
+
     public function __construct()
     {
         $this->postService = new PostService;
     }
-    
+
     /**
      * Create user posts
      */
     public function create(PostRequest $request): RedirectResponse
     {
-        $userId = auth()->id(); 
-        $posts = $this->postService->create($request, $userId);
-        
+        $userId = auth()->id();
+        $post = $this->postService->create($request, $userId);
+        $this->postService->addPhoto($request, $userId, $post);
```

Comment:
This is multiple images. it should be addPhotos. It must be plural if the method is doing multiple requests.